infrastructure/templates/public-api/components/wafPolicy.bicep

```diff
@@ -0,0 +1,40 @@
+@description('Specifies the location for all resources')
+param location string
+
+@description('Specifies the name of the policy')
+param name string
+
+@description('Specifies a set of tags with which to tag the resource in Azure')
+param tagValues object
+
+resource policy 'Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies@2023-11-01' = {
+  name: name
+  location: location
+  properties: {
+    policySettings: {
+      requestBodyCheck: true
+      maxRequestBodySizeInKb: 128
+      fileUploadLimitInMb: 100
+      state: 'Enabled'
+      mode: 'Prevention'
+      requestBodyInspectLimitInKB: 128
+      fileUploadEnforcement: true
+      requestBodyEnforcement: true
+    }
+    managedRules: {
+      managedRuleSets: [
+        {
+          ruleSetType: 'OWASP'
+          ruleSetVersion: '3.2'
+        }
+        {
+          ruleSetType: 'Microsoft_BotManagerRuleSet'
+          ruleSetVersion: '0.1'
```

Comment:
should this be `1.0`?

Reply:
This is what was extracted from the templates that were set up when I created the App Gateway manually, so this is what would have been set via the UI.  I'm happy to update and try!

Updated ta - and great spot!

Reply:
no worries! I had a look and it seemed that 1.0 was the latest :)